website/content/docs/k8s/observability/metrics.mdx

```diff
@@ -76,4 +74,21 @@ global:
 
 ## Metrics in the UI Topology Visualization
 
+Consul’s built in UI has a topology visualization for services part of the Consul-Connect service mesh. The topology visualization has the ability to fetch basic metrics from a metrics provider for each service and display those metrics as part of the topology visualization.
+
+The diagram below illustrates how the UI display service metrics for a sample application:
+
+[![Metrics Merging Architecture](/img/ui-topology.png)](/img/ui-topology.png)
+
+This could previously be configured by previously setting up `server.extraConfig` but can now be configured using fields under `ui.metrics`. This will enable the Consul UI to query the provider specified by `ui.metrics.provider` at the baseURL `ui.metrics.baseURL` to display
```

Comment:
```suggestion
This could previously be configured via `server.extraConfig` but can now be configured under `ui.metrics`. This will enable the Consul UI to query the provider specified by `ui.metrics.provider` at the URL of the Prometheus server `ui.metrics.baseURL` to display
```